business_requirement_deliverable/models/business.py

```diff
@@ -49,6 +50,40 @@ class BusinessRequirementResource(models.Model):
         string='Business Requirement Deliverable',
         ondelete='cascade'
     )
+    business_requirement_id = fields.Many2one(
+        comodel_name='business.requirement',
+        string='Business Requirement',
+        required=True,
+    )
+    business_requirement_partner_id = fields.Many2one(
+        comodel_name='res.partner',
+        related='business_requirement_id.partner_id',
+        string='Business Requirement',
+        store=True
+    )
+    business_requirement_project_id = fields.Many2one(
+        comodel_name='project.project',
+        related='business_requirement_id.project_id',
+        string='Business Requirement',
+        store=True
+    )
+
+    @api.model
+    def fields_view_get(self, view_id=None,
+                        view_type='form', toolbar=False, submenu=False):
+        result = super(BusinessRequirementResource,
```

Comment:
Add docstring saying for what is this needed. Reading code, I don't see this very clear and with possible side effects, making editable a tree based on the active_id (which I suppose that can be changed navigating through records).